src/util/saveArticles.ts

```diff
@@ -249,32 +231,35 @@ export function getArticleUrl(downloader: Downloader, dump: Dump, articleId: str
 /*
  * Fetch Articles
  */
-export async function saveArticles(zimCreator: ZimCreator, downloader: Downloader, dump: Dump, forceRender = null) {
+export async function saveArticles(zimCreator: ZimCreator, downloader: Downloader, dump: Dump, hasWikimediaMobileApi: boolean, forceRender = null) {
   const jsModuleDependencies = new Set<string>()
   const cssModuleDependencies = new Set<string>()
+  const jsMobileModuleDependencies = new Set<string>()
+  const cssMobileModuleDependencies = new Set<string>()
```

Comment:
Why are they declared separatly? Why not pushing what needs to be pushed in `jsModuleDependencies` and `cssModuleDependencies` ?

Reply:
During my first iteration of PCS module implementation, I separated them into distinct variables to avoid side effects and reduce the complexity of the refactoring process. Now it is updated regarding your requirements.